forte/data/data_store.py

```diff
@@ -64,18 +65,25 @@ def __init__(self, onto_file_path: Optional[str] = None):
         <attr_n>].
         A group type entry has the following format:
         [<member_type>, <[members_tid_list]>, <tid>, <entry_type>, <attr_1>,
-            <attr_2>, ..., <attr_n>].
+            <attr_2>, ..., <attr_n>, index_id].
```

Comment:
Yes in line 83 `It is an extra field to record the location of the entry in the list. When the user adds a new entry to the data store, the `index_id` will be created and appended to the end of the original `entry data` list.` 

I want to add an extra field in the data so that we can have the index when we have tid: 
entry_dict[tid][-1].

Then, we use this index in delete because we also need to remove data from the list.